darkcluster/src/main/java/com/linkedin/darkcluster/api/DarkClusterResponseValidationMetricsCollector.java

```diff
@@ -0,0 +1,24 @@
+package com.linkedin.darkcluster.api;
+
+import java.util.Map;
+
+/**
+ * Collector of incoming response validation metrics from dispatchers.
+ * This is meant to be called on the dark cluster hosts where the request headers containing response validation metrics are read.
+ * A dark cluster host may receive metrics from multiple dispatchers and this interface defines method for collecting these
+ * incoming metrics.
```

Comment:
this is good, but lets also add a sentence saying why this is a useful feature to have. Something like:
"This allows response comparison metrics about a dark cluster to be displayed (via a monitoring/graphing system) with all other system metrics in a dark cluster host. These response comparison metrics are not specific to this dark cluster host, but that should be sufficient for most cases, and further granularity can be achieved by having separate dark clusters.